recipes/great-expectations/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "great-expectations" %}
+{% set version = "0.3.2" %}
+{% set sha256 = "30d14050cb48ea45238ade33b6ec027e44f6293d88ce140c2e278c47bee4b79e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|replace("-","_") }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # Upstream setup.py read docs folder to pickup the long-description
+    - 00-force-long_description.patch
+
+build:
+  #noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.9.2
+    - scipy >=0.19.0
+    - pandas >=0.20.3
+    - python-dateutil >=2.4.2
+    - pytz >=2015.6
+    - argh >=0.26.2
+    - jsonschema >=2.5.1
+
+test:
+  imports:
+    - great_expectations
+
+about:
+  home: https://github.com/great-expectations/great_expectations
+  license: Apache-2.0
+  license_family: Apache
+  # License is still missing
+  #license_file: LICENSE
```

Comment:
It would be probably necessary to [package the LICENSE file manually](https://conda-forge.org/docs/meta.html#packaging-the-license-manually) (it's a requirement for Apache License).

Reply:
It would be nice if this could be addressed upstream..